Description: use python3 interpreter, not python by default in Makefile
  The rest of the original patch by Steve Langasek <steve.langasek@ubuntu.com>
  are no longer needed/adopted upstream
Last-Modified: 2019-09-06

--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # simple makefile to simplify repetetive build env management tasks under posix
 # Ideas borrowed from scikit-learn's and PyMVPA Makefiles  -- thanks!
 
-PYTHON ?= python
+PYTHON ?= python3
 
 MODULE ?= datalad
 
